cmd/influxd/server_integration_test.go

```diff
@@ -197,8 +198,13 @@ func query(t *testing.T, nodes Cluster, urlDb, query, expected string) (string,
 			t.Fatalf("Couldn't read body of response: %s", err.Error())
 		}
 
-		if expected != string(body) {
+		if expected != "" && expected != string(body) {
```

Comment:
We should update the comment, explaining that `expected` is ignored if set to `""`.